met-web/src/components/engagement/form/EngagementWidgets/Documents/DocumentOptionCard.tsx

```diff
@@ -71,7 +71,7 @@ const DocumentOptionCard = () => {
                         spacing={1}
                     >
                         <Grid item>
-                            <DescriptionOutlinedIcon sx={{ pl: '25px', fontSize: '5em' }} />
+                            <DescriptionOutlinedIcon htmlColor={'#707070'} sx={{ pl: '25px', fontSize: '5em' }} />
```

Comment:
Can we take if from here
![image](https://user-images.githubusercontent.com/91914654/216394280-4ab6780b-0782-476c-a86e-3630fcaefe83.png)

Reply:
done updated the info too & added it to palette thank you